WTG.Analyzers.Utils/ExpressionRemover.cs

```diff
@@ -46,26 +46,21 @@ public override SyntaxNode VisitParenthesizedExpression(ParenthesizedExpressionS
 
 			public override SyntaxNode VisitBinaryExpression(BinaryExpressionSyntax node)
 			{
-				switch (node.Kind())
+				return (node.Kind()) switch
```

Comment:
why do all these new switch expressions have outer parentheses? I don't think they're needed.

Reply:
The parentheses were added by the code-fix that converted from the switch statement to switch expression and nothing complained about them being superfluous. I have gone through and removed them.

Reply:
Hmm, I'm tempted to escalate that as a bug to Roslyn.